documentation/doxygen/extract_docstrings.py

```diff
@@ -64,6 +64,8 @@ def run_fast_scandir(dir, ext):    # dir: str, ext: list
     module = ast.parse(file_contents)
     ds = ast.get_docstring(module)
     if ds is not None:
-        with open(pyz_file_path + '.pyzdoc', 'w') as pyz_doc_file:
+        pyz_filename = pyz_file_path
+        pyz_filename = pyz_dir_out + '/' + pyz_filename[pyz_filename.rfind("_") : : ] + 'zdoc'
+        with open(pyz_filename, 'w') as pyz_doc_file:
```

Comment:
```suggestion
        pyz_filename_out = pyz_dir_out + '/' + pyz_file_path[pyz_file_path.rfind("_"):] + 'zdoc'
        with open(pyz_filename_out, 'w') as pyz_doc_file:
```

Reply:
In addition: there is still some comment missing to explain the new concatenation of `zdoc`.